ocs_ingester/fits.py

```diff
@@ -12,6 +12,15 @@
 
 logger = logging.getLogger('ocs_ingester')
 
+LETTER_TO_OBSTYPE = {
+    'b': 'BIAS',
```

Comment:
Do you want to add 'a' to be ARC? Also how does this work if there are multiple types for FLOYDS vs NRES?

Reply:
I handle 'e' and 'a' in the method in the code, since they have different types based on if we think its NRES, FLOYDS, or anything else